webrtc.html

```diff
@@ -648,6 +657,39 @@ <h4><dfn>RTCRtcpMuxPolicy</dfn> Enum</h4>
           </div>
         </div>
       </section>
+      <section>
+        <h4><dfn>RTCAdaptivePtime</dfn> Enum</h4>
```

Comment:
Why bother with an enum? Is there any chance we'll ever want to do something different from "true" and "false"?

Reply:
> Why bother with an enum? Is there any chance we'll ever want to do something different from "true" and "false"?

I understood Philip Jägenstedt as wanting that. A boolean would serve my needs equally well.